samples/extensions/multipleroutes/README.md

```diff
@@ -0,0 +1,3 @@
+This extension, once installed, will disable multiple routes option in WebRTC. This will prevent any website from accessing local machine's private IP addresses or public ones which should have been hidden by the usage of VPN. The only address left for WebRTC will be the same one used for http traffic.
```

Comment:
"will disable multiple routes option in WebRTC"  might be better as "will disable the option for multiple network routes in WebRTC